app/controllers/idv/capture_doc_status_controller.rb

```diff
@@ -11,26 +11,34 @@ def show
     private
 
     def document_capture_session_poll_render_result
-      return { plain: 'Unauthorized', status: :unauthorized } unless flow_session
-      session_uuid = flow_session[:document_capture_session_uuid]
-      document_capture_session = DocumentCaptureSession.find_by(uuid: session_uuid)
-      return { plain: 'Unauthorized', status: :unauthorized } unless document_capture_session
-      return { plain: 'Cancelled', status: :gone } if document_capture_session.cancelled_at
-
-      render_result(document_capture_session)
-    end
+      return { json: nil, status: :unauthorized } unless flow_session
+      return { json: nil, status: :unauthorized } unless document_capture_session
+      return { json: nil, status: :gone } if document_capture_session.cancelled_at
+      return {
+        json: { redirect: idv_session_errors_throttled_url },
+        status: :too_many_requests,
+      } if is_throttled
```

Comment:
let's name this with a `?`

```suggestion
      } if throttled?
```

Reply:
> let's name this with a `?`

Yep, good call. Updated in b5ca39d08.